docs/topics/circuit-breaker-mission-intro-paragraph.adoc

```diff
@@ -1,3 +1,6 @@
-Mission proficiency level: xref:proficiency_foundational[*{proficiency-foundational}*].
+Mission proficiency level: 
+//special case since topic is used by front end.
+ifdef::docs-topic[xref:proficiency_foundational[*{proficiency-foundational}*].]
+ifndef::docs-topic[link:https://appdev.openshift.io/docs/wf-swarm-runtime.html#proficiency_levels[*Foundational*].]
```

Comment:
@tradej it's defined here: https://github.com/openshiftio/appdev-documentation/blob/2abfe1aaa7f82fb2ed885bcc5e6c301ec28224f7/docs/topics/templates/document-attributes.adoc#L132 and that works
![screenshot from 2017-08-17 13-26-32](https://user-images.githubusercontent.com/51133/29410194-cf6ab824-834f-11e7-9224-ab6ccd5b2aa4.png)